js/script.js

```diff
@@ -1,290 +1,127 @@
+//require("modernizr");
+
+var $ = require("jquery");
+var _ = require("lodash");
+var openstates = require("./openstates");
+var capitalize = require("./capitalize");
+var templates = require("./templates");
+var qs = require("querystring");
+
+//alias for Foundation
+window.jQuery = window.$ = $;
+
+//state variables
+var currentCommittee = null;
+
+var getQueryVar = function(name) {
+  var query = qs.parse(window.location.search.replace(/^\?/, ""));
+  return query[name];
+};
+
 // Get names of the committees
 var populateDropDown = function(state) {
-  var committeeListRequest = $.ajax({
-    dataType: "json",
-    url: "http://openstates.org/api/v1/committees/",
-    data: {
-      state: state,
-      fields: 'members,committee,subcommittee,chamber',
-      apikey: "9e3e71730ae34e1ebbf4dd0e1c346c07"
-    }
-  })
-    .done(function(committeeList) {
-      committees = _.reject(committeeList, 'subcommittee');
```

Comment:
Each time you call a `_` function, it's another loop behind the scenes. That means this version (which calls `reject` once and `where` twice) is at least three times as slow as the loop you'll see in my version. However, I think you should err on the side of readability over speed (which is not really a huge problem in the number of results you're likely pulling), and if you find this version easier to follow, keep your code.